go.mod

```diff
@@ -65,7 +65,6 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.15.0
-	github.com/spyzhov/ajson v0.7.2
```

Comment:
Given https://github.com/spyzhov/ajson/issues/63 seems positive about merging this upstream, would it be preferable here to use a go mod replacement instead of a hard fork? We can point the replacement at the fork for now until it's part of upstream.

That also makes it easier to do the actual PR upstream since you don't need to rename packages etc. then there as well in our fork.

Reply:
Good idea, will make the change now.